src/arrayMethodJoin.js

```diff
@@ -4,8 +4,24 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let result = '';
+    const lastElement = this.length - 1;
```

Comment:
[useful link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at)

```suggestion
    const lastElement = this.at(-1)
```

Reply:
I tried, but VSCode says "TypeError:` this.at` is not a function". In Slack, another mentor said that the reason of this error is that  the tests don't know about `at()` method.